### ISSUE TITLE
After Success Updating Data then The variable will be filled with the random data

### ISSUE THREAD

**Issue by gieart87** [chr15k/laravel-mysql-encrypt#3]
Hi @chr15k
Thanks for sharing this cool package. It works perfectly until I found some strange issue like the following details.

Model:
```
class Agent extends Model
{
    use HasFactory, HasUuids, SoftDeletes, Encryptable;

    protected $encryptable = [
        'phone',
        'email',
        'address1',
        'address2'
    ];
}
```
Assumed that I have an `$agent` variable contains data with the certain ID like this.
```
$agent = Agent::findOrFail('xxxx');
```

I updated one of the encryptable fields (e.g. `phone`).

```
$agent->phone = $this->phone;
$agent->save();
```

I printed the `$agent` variable after the `save()` method. Surprisingly I found random agent data with different ID and usually the first row on the table.

Any suggestion regarding the issue would be appreciated.

Thanks

**Comment by chr15k** [comment]
Hi @gieart87 well spotted, I can see what the problem is, and will push a hotfix shortly. Thanks raising this issue!

**Comment by chr15k** [comment]
Hi @gieart87 can you please pull the latest [2.x] (2.0.4) and re-test? Thanks!

**Comment by chr15k** [comment]
Closing as this issue should be resolved as of 2.0.4
